src/components/Layout/SaasSelector.tsx

```diff
@@ -73,13 +71,9 @@ export const SelectCustom = ({
   return (
     <FormControl isInvalid={!!errors[name]}>
       {label && (
-        <FormLabel htmlFor={name} display='flex' ms={1} fontSize='sm' fontWeight='500' color={textColor} mb={2}>
+        <FormLabel htmlFor={name} display='flex' ms={1} fontSize='sm' fontWeight='500' mb={2}>
           {label}
-          {required && (
-            <Text color={textColorBrand} ml={1}>
-              *
-            </Text>
-          )}
+          {required && <Text ml={1}>*</Text>}
```

Comment:
As said before, there's no need to manually add the `*` on each field. Use the `isRequired` prop instead.